lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -326,10 +326,15 @@ enum PaymentError {
     "SignerError",
 };
 
+[Enum]
+interface BlockchainExplorer {
+    Electrum(string url);
+    Esplora(string url, boolean use_waterfalls);
```

Comment:
Esplora won't work well on the sdk since it lacks the batching option. IMO we can go with Waterfall variant instead.